README.md

```diff
@@ -16,42 +16,38 @@ API service and web client of the Registry of Open Community Challenges
 
 1. Create the file that contains the future environment variables
 
-       cp .env.sample .env
+       cp .env.example .env
```

Comment:
Would it be helpful to explicitly mention that the file will work as-is after copying?  If they _wanted_ to, they can edit the environment variables, but it is not necessary.

Reply:
Done in next commit.